azkaban-common/src/main/java/azkaban/utils/EmailMessage.java

```diff
@@ -287,4 +287,13 @@ public EmailMessage println(Object str) {
 
     return this;
   }
+
+  public String getBody() {
```

Comment:
> This method is for unit testing only, right?

Yes. But no harm done if someone calls them in actual application code.

> How about making it package private? This way the readers would know that this is not a public API outside this package and would reduce the search space.

Can't use empty/default package because different packages:
`azkaban.utils.EmailMessage`
`azkaban.executor.mail.DefaultMailCreator`

In this case, I hope it's acceptable to add these getters.